src/screens/Details/Details.styles.ts

```diff
@@ -40,20 +40,20 @@ export default StyleSheet.create<Style>({
   },
   artistName: {
     padding: padding.eighth,
-    fontSize: padding.base,
+    fontSize: fontSizes[0],
```

Comment:
Magic indexes lead to unreadable code. Could this be a map with ENUM keys like "SMALL" "MEDIUM" "LARGE" "X-LARGE"

This way if you wanted to add an X-SMALL (something smaller than fontSizes[0], you wouldn't have to reindex all references as +1

Reply:
I like that, tbh I haven't used TS enums yet but this defs fits that use case.